### ISSUE TITLE
Overhaul user's first experience

### ISSUE THREAD

**Issue by dannon** [galaxyproject/galaxy#3950]
Need an overhauled first user experience.

This would be a full-page first-time-user welcome, separate from the existing welcome.html.

![image](https://cloud.githubusercontent.com/assets/155398/25240765/b3a6766e-25c1-11e7-95e6-736c84bb2583.png)

**Comment by martenson** [comment]
Displays to anonymous users or users that do not own a dataset. Also reachable from Help menu.

**Comment by hexylena** [comment]
Is this still on the roadmap? It sounds really nice!

**Comment by martenson** [comment]
xref fairly old attempt: https://github.com/galaxyproject/usegalaxy-playbook/pull/166

**Comment by shiltemann** [comment]
@astrovsky01 this sounds great, feel free to find me on Gitter to discuss your ideas for this. Our idea on the GTN was also to have several "microtutorials" that are just a very short guide through a Galaxy feature, so seems like that might fit here as well :+1:

**Comment by shiltemann** [comment]
Another thought: we could also have a page like this at first login after every release update of Galaxy, highlighting the new features of Galaxy? Since @hexylena and others have been making nice user-facing release notes, and community members like GarethPrice-Aus have made  videos highlighting new features, this could be a good way to make these more findable for end users?

**Comment by hexylena** [comment]
Yeah! @shiltemann suggested this in the elixir galaxy telco and it sounded fantastic. SO many applications have a popup of "Hey, welcome, we've updated" that must be dismissed before they can continue. This would be a great place to stick the existing user-facing release notes I've been building.

It might need something like a central notification system (#1023/#2324) to ensure that users don't see this repeatedly across browsers? But it would be such an improvement to communicate with our users what's changed.

**Comment by nekrut** [comment]
@jmchilton can you please ask @astrovsky01 to present this as a Dev Roundtable in two weeks (after @guerler)

**Comment by astrovsky01** [comment]
The 13th?

**Comment by jmchilton** [comment]
Yup, does that work for you?

**Comment by astrovsky01** [comment]
Yeah, sounds good,

**Comment by carolg** [comment]
Sounds really great to have a separate welcome page. For the future Usegalaxy.ca we were wondering how we could implement something in that line with the current galaxy config. So for the moment we are looking to have our entry page at a different URL. Usegalaxy.ca will show static content alone, the actual galaxy will be at something like app.usegalaxy.ca.
You can actually see at https://usegalaxy.ca. We tried to follow the style of the ComunityHub site. Login button leads to nowhere for now.

**Comment by carolg** [comment]
Just realised this is a really old thread :(. Where should I post this ?

**Comment by martenson** [comment]
@carolg check out the different landing pages for the various subdomains usegalaxy.eu hosts: https://galaxyproject.org/eu/subdomains/

This is what is usually done for customization of a specific Galaxy landing page. If you mean specifically the new user welcome/experience that has been implemented in https://github.com/galaxyproject/galaxy/pull/11085 (so we could likely close this issue?)

**Comment by carolg** [comment]
Thanks for the links @martenson 
I realized I mixed up the first time "user" as opposed to the Landing page for first time "visitor." So my comment is definitely not at the right place :). It was more a point of information for discussion, no real issue. Thanks. I'll go on Matrix.
